### ISSUE TITLE
Add AI controlled "drone" cars to chase

### ISSUE THREAD

**Issue by CurryKitten** [CurryKitten/CurryKitten-Sim#9]
Chasing things is both fun and a useful skill to practice/master.  Add the option to chase another quad and car within the sim.  Potentially tie this in with the interactive lessons as a how to, but always have the option for a user to "freestyle" and just have the vehicles available to chase around.

**Comment by CurryKitten** [comment]
I've modified the issue to specifically reflect cars and added new issues for planes #29 and quads #30 as the control and AI methods of implementing them are quite different, although they'll all get initiated form the Scenarios menu.

**Comment by CurryKitten** [comment]
Cars have been implemented and will be released in 0.42a.  To facilitate this I've added in a Sceanrios menu (where in the future other activities will be launched from)

![Screenshot 2020-01-29 at 21 09 49](https://user-images.githubusercontent.com/12443664/73436823-1dcc0080-4343-11ea-9e30-87beb9923679.jpg)

I created the ability to use between 1 and 3 cars at a range of speeds in 3 different circuit configurations.  Each of the cars has subtly different qualities in order to make it look less computer controlled and a little more random.  I could support more cars (for added chaos) but it would be useful to get some feedback from people on low-spec machines and make sure there wasn't a problem in this initial release
